app/views/reports/itt_new_starter_data_signoff.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= render PageTitle::View.new(text: "Reports") %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLinkComponent.new(
+    text: t("back"),
+    href: root_path,
+    ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class = govuk-grid-column-two-thirds-from-desktop>
+  <h1 class="govuk-heading-l">Export new trainee data for sign off for the <%= AcademicCycle.current.label %> academic year</h1>
+    <p class="govuk-body">This export will show the new trainee data you’ve provided to the DfE for trainees who’ve started their initial teacher training (ITT) in the <%= AcademicCycle.current.label %> academic year, and have a trainee start date on or before the second Wednesday of October <%= AcademicCycle.current.start_year %> (sometimes called the census date).</p>
+    <div class="govuk-inset-text">For the <%= AcademicCycle.current.label %> academic year, the ITT census date is <%= @census_date %>.</div>
+    <p class="govuk-heading-m">What you must do to sign off your new trainee data</p>
+    <p class="govuk-body">Use this export to check you’ve provided accurate data for all your new trainees. You should check for any errors or missing trainees.</p>
+    <p class="govuk-body">Once you’ve checked your data, a senior person from your organisation (preferably a different person to who submitted the data) should sign it off <a href="#" class="govuk-link">using this form</a>.</p>
```

Comment:
@MaeveRoseveare @claire-hughez this references the report here but there's no link. Should we add the link now, or else reword this bit?

Reply:
Yes, just discussed with @claire-hughez and let's add it.

Reply:
I've added the link as per maeves comment